.github/workflows/linux_builds.yml

```diff
@@ -199,9 +199,9 @@ jobs:
       - name: Check for extension api updates
         if: ${{ matrix.godot-cpp-test }}
         run: |
-          echo "Running --dump-extension-api to create extensions api."
-          VK_ICD_FILENAMES=$(pwd)/vk_swiftshader_icd.json DRI_PRIME=0 xvfb-run ${{ matrix.bin }} --audio-driver Dummy --dump-extension-api 2>&1 > /dev/null || true
-          misc/scripts/compare_extension_api.py godot-cpp/godot-headers/extension_api.json extension_api.json
+          echo "Running --dump-extension-api and --dump-gdnative-interface to create extensions api."
+          VK_ICD_FILENAMES=$(pwd)/vk_swiftshader_icd.json DRI_PRIME=0 xvfb-run ${{ matrix.bin }} --audio-driver Dummy --dump-extension-api --dump-gdnative-interface 2>&1 > /dev/null || true
+          misc/scripts/compare_extension_api.py godot-cpp/godot-headers/extension_api.json extension_api.json godot-cpp/godot-headers/godot/gdnative_interface.h gdnative_interface.h
```

Comment:
For now this script is not implemented, but it's probably not so useful to compare `gdnative_interface.h` per se, as it's not generated like the API.

The aim with that script would be to spot whenever we unintentionally break compatibility with changing parts of the API. For `gdnative_interface.h` itself, if we're breaking compatibility this would be present in the PR itself and it should be fairly easy to notice.

Reply:
Ok, I will remove that change then. I was not sure it would be useful or not.